Simplified/Book/Models/NYPLBook.m

```diff
@@ -618,4 +618,19 @@ - (NYPLBookContentType)defaultBookContentType
   return defaultType;
 }
 
+- (void)addCustomExpirateDate:(nonnull NSDate *)date
+{
+  if (self.defaultAcquisitionIfOpenAccess.type &&
```

Comment:
`self.defaultAcquisitionIfOpenAccess.type &&` can be safely removed: if `self.defaultAcquisitionIfOpenAccess.type` is nil, `[self.defaultAcquisitionIfOpenAccess.type isEqualToString:ContentTypeAxis360]` will always fail.